components/ILIAS/Authentication/classes/Setup/class.ilSessionMaxIdleIsSetObjective.php

```diff
@@ -136,7 +136,21 @@ public function achieve(Setup\Environment $environment): Setup\Environment
 
     public function isApplicable(Setup\Environment $environment): bool
     {
-        return true;
+        set_error_handler(static function (int $severity, string $message, string $file, int $line): never {
+            throw new ErrorException($message, $severity, $severity, $file, $line);
+        });
+
+        try {
+            $conn = fsockopen("localhost", 80);
```

Comment:
Thanks @daniwe4 for the change.

One more remark: You suggest to create a socket connection to "localhost" here, but in `achive`, we read the URL (with the host we want to connect to) from the INI file. Should we extract the hostname from the INI value?

Reply:
I just tried the URL. Unfortunately that doesn't seem to work. Do you know how that works? 
Otherwise it's just a check whether a web server is running while the setup is being carried out. We could still pull http or https from the URL and then adjust the port.

Reply:
Hello @mjansenDatabay ,

I revised everything again.

Greetings Danie